src/blocks/weather/open_weather_map.rs

```diff
@@ -1,13 +1,15 @@
 use super::*;
 use chrono::Utc;
+use serde::{de, Deserializer};
 
-pub(super) const URL: &str = "https://api.openweathermap.org/data/2.5/weather";
+pub(super) const CURRENT_URL: &str = "https://api.openweathermap.org/data/2.5/weather";
```

Comment:
#1267 mentions using /onecall, but this requires a different api key.

Reply:
Both of these call still allow city_id and place still, but #1485 should be tackled next.